manual on codes draft (#359)
wmo-im/iwxxm: documentation/manual/FM205.adoc

(j) *Quantitative Volcanic Ash (QVA) Concentration Information* is a message providing information on the concentration of volcanic ash in the atmosphere, including the vertical extent and horizontal distribution of the ash.

(k) *Volcano Observatory Notice for Aviation (VONA)* is a message issued by a volcano observatory to provide information on volcanic activity, including eruptions, ash plumes, and other related phenomena.

== 205-2025-2.2 IWXXM versions and amendments

*205-2025-2.2.1* After IWXXM 3.0, the IWXXM version is represented by the release year and a number.

*205-2025-2.2.2* Each release of IWXXM is composed of individual packages with their version number following the semantic versioning: MAJOR.MINOR.PATCH (example TAF 3.0.1) where: 
– A MAJOR (X.y.z) version introduces major conceptual changes. Forward data mapping is not guaranteed. 
– A MINOR (x.Y.x) version introduces new model elements and capabilities guaranteeing forward data mapping for non-deprecated elements or deprecated elements with replacement. Loss of information may happen for elements that are no longer required.

> – A PATCH (x.y.Z) version is limited to bug fixing. Forward and backward data mapping is guaranteed without loss of information. A PATCH version does not impose changes in a consuming system. A PATCH version may still require data conversion to be done ahead of the data entering the consuming system (done by a mediator service).

Author: What does this mean? Is there another way to write it? 
> A PATCH version may still require data conversion to be done ahead of the data entering the consuming system (done by a mediator service).

> – A PATCH (x.y.Z) version is limited to bug fixing. Forward and backward data mapping is guaranteed without loss of information. A PATCH version does not impose changes in a consuming system. A PATCH version may still require data conversion to be done ahead of the data entering the consuming system (done by a mediator service).

Author: ```suggestion
– A PATCH (x.y.Z) version is limited to bug fixing. Forward and backward data mapping is guaranteed without loss of information. A PATCH version does not impose changes in a consuming system, however,  data conversion may still be required.
```

– A MINOR (x.Y.x) version introduces new model elements and capabilities guaranteeing forward data mapping for non-deprecated elements or deprecated elements with replacement. Loss of information may happen for elements that are no longer required.
– A PATCH (x.y.Z) version is limited to bug fixing. Forward and backward data mapping is guaranteed without loss of information. A PATCH version does not impose changes in a consuming system; however,  data conversion may still be required.

*205-2025-2.2.3* Each IWXXM version addresses the requirements of amendments to ICAO Annex 3 and/or PANS-MET. Those amendments affect the packages composing IWXXM in different ways, and the resulting new version of the packages shall reflect the level of change required by the amendments following the semantic versioning described in 205-2025-2.2.2.

*205-2025-2.2.4* The following table identifies which version of each package composing IWXXM complies with specific ICAO Annex 3 and PANS-MET amendments.

[options="header"]
|===
|ICAO Annex 3 Amendment|78|79-80|79-81|82
|*PANS-MET Edition*|n/a|n/a|n/a|*first*

> |IWXXM Version|3.0.0|2021-2|2023-1|2025-2

Author: should we include common and metFeature in this table?

|State of the sea |http://codes.wmo.int/bufr4/codeflag/0-22-061
|Airframe icing |http://codes.wmo.int/bufr4/codeflag/0-20-041
|===

*205-2025-2.4.5* Codes for use with IWXXM Meteorological Feature Package and its extension packages
[width="100%",cols="20%,43%,37%",options="header",]
|===
|Name |Description |URL

> |Nil reasons |Nil-reason terms are used to provide an explanation for recording a missing (or void) value within a data product. Terms are drawn from authorities in addition to WMO,including ISO/TC 211 (from ISO 19136:2007 Geography markup language clause 8.2.3.1; published on their behalf by the Open Geospatial Consortium).	 |http://codes.wmo.int/iwxxm/nil

Author: ```suggestion
|Nil reasons |Nil-reason terms are used to provide an explanation for recording a missing (or void) value within a data product. Terms are drawn from authorities in addition to WMO, including ISO/TC 211 (from ISO 19136:2007 Geography markup language clause 8.2.3.1; published on their behalf by the Open Geospatial Consortium).	 |http://codes.wmo.int/iwxxm/nil
```
